packages/gatsby/src/state-machines/waiting/index.ts

```diff
@@ -0,0 +1,110 @@
+import { MachineConfig, assign, Machine } from "xstate"
+import { IWaitingContext } from "./types"
+import { waitingActions } from "./actions"
+import { waitingServices } from "./services"
+
+const NODE_MUTATION_BATCH_SIZE = 100
```

Comment:
Please add comments on why you chose these numbers.

Reply:
`magic.gif`. 

They're entirely arbitrary, and just seemed to work ok. This should be configurable, but with sensible defaults.

Reply:
that's fine :) just a comment would help here that they are randomly chosen.

Reply:
Will do